### PR TITLE
Check for PostNord order in bulk action

### PR DESCRIPTION
**Pull request by samaij**
image

### CODE REVIEW

File: src/Service/VgPostnordBookingService.php [hunk 1]
```diff
@@ -64,21 +64,15 @@ public function createBlankBooking(int $id_order): VgPostnordBooking
...
-                $mandatory_services = $carrier_settings[$carrier->id_reference]["mandatory_service_codes"] ?? [];
-            }
-            if (array_key_exists("additional_service_codes", $carrier_settings[$carrier->id_reference])) {
-                $additional_services = $carrier_settings[$carrier->id_reference]["additional_service_codes"] ?? [];
-            }
+            $carrier_config = $carrier_settings[$carrier->id_reference];
+            $additional_service_codes = \Vg_postnord::getCombinedServiceCodesForConfig($carrier_config);
         }
-        $additional_service_codes = implode(",", array_unique(array_merge($mandatory_services, $additional_services)));
+        $additional_service_codes = implode(",", $additional_service_codes);
```
Comment (tswfi) [inline src/Service/VgPostnordBookingService.php:75]: getCombinedServiceCodesForConfig already returns an array?

File: vg_postnord.php [hunk 2]
```diff
@@ -1349,12 +1349,12 @@ public function hookActionValidateOrder(array $params)
...
      * @return bool
      */
-    private function isPostNordOrder(int $id_order): bool
+    public static function isPostNordOrder(int $id_order): bool
     {
         try {
             $order = new Order($id_order);
         } catch (PrestaShopException $e) {
-            $this->logger->error('Error loading Product', [
+            static::getLogger()->error('Error loading Product', [
```
Comment (tswfi) [inline vg_postnord.php:1357]:
```suggestion
            static::getLogger()->error('Error loading Order', [
```

File: vg_postnord.php [hunk 3]
```diff
@@ -1363,7 +1363,7 @@ private function isPostNordOrder(int $id_order): bool
         }
 
         $carrier = new Carrier($order->id_carrier);
-        return $carrier->external_module_name === $this->name;
+        return $carrier->external_module_name === "vg_postnord";
```
[the remark this answers is not in the document]
Comment (samaij) [reply vg_postnord.php]: A great one even, feels like I completely forgot that constants exist :D

https://github.com/vilkasgroup/vg_postnord/pull/17/commits/280d41836e1e90942b9160ed6033b82e1eb9112a

File: src/Service/VgPostnordBookingService.php (hunk 1 above)
Comment (samaij) [reply src/Service/VgPostnordBookingService.php:75]: Yes.

**Review by tswfi — APPROVED** [review]

**Pull request merged**